src/drive/styles/filelist.styl

```diff
@@ -372,10 +372,14 @@ column-width-thumbnail-bigger = 7rem
         flex 1 1 auto
 
     .fil-content-header-action
+        flex 0
+        box-sizing content-box
```

Comment:
You can add `width 100%` so container and icon take the same place because there is an `width 1.5rem` set by default for both header and file.